fdbrpc/FlowTransport.actor.cpp

```diff
@@ -1029,22 +1021,24 @@ ACTOR static void deliver(TransportData* self,
 	}
 
 	auto receiver = self->endpoints.get(destination.token);
-	if (receiver && (authorizedTenants->trusted || receiver->isPublic())) {
+	if (receiver && (isTrustedPeer || receiver->isPublic())) {
 		if (!checkCompatible(receiver->peerCompatibilityPolicy(), reader.protocolVersion())) {
 			return;
 		}
 		try {
 			g_currentDeliveryPeerAddress = destination.addresses;
+			g_currentDeliverPeerAddressTrusted = isTrustedPeer;
```

Comment:
Currently there doesn't seem to be any receivers that invoke actor functions.
However, it would be nice to have a no-actor-below-this-scope guarantee in place such as this.

Reply:
Maybe we could add an assert for now that these parameters are in an unset state when we enter this block

Reply:
Done!